db/seeds.rb

```diff
@@ -6,10 +6,10 @@
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
 # user = User.create(:id)
-user3 = User.create!(id:3, email:"email3@email.com", role: "member", password:"123456")
-user4 = User.create!(id:4, email:"email4@email.com", role: "member", password:"123456")
-user5 = User.create!(id:5, email:"email5@email.com", role: "member", password:"123456")
-user6 = User.create!(id:6, email:"email6@email.com", role: "member", password:"123456")
+user3 = User.create!(id:7, email:"email3@email.com", role: "member", password:"123456")
+user4 = User.create!(id:8, email:"email4@email.com", role: "member", password:"123456")
+user5 = User.create!(id:9, email:"email5@email.com", role: "member", password:"123456")
+user6 = User.create!(id:10, email:"email6@email.com", role: "member", password:"123456")
```

Comment:
an `id` is not required to create a user - ActiveRecord will assign an id upon creation